docs/setup/vbmh-setup.md

```diff
@@ -0,0 +1,203 @@
+```
+IP_ADDR=$(ifconfig | grep "bond0: " -A 1 | awk '/inet / {print $2}')
+```
+- This grep command is specific to the network configuration typically found on an Equinix server, on other environments you will need to manually add your ip address to the `IP_ADDR` variable for future commands.
+
+Grab the mac addresses and Sushy-Tools IDs of the virtual machines and save in variables
+```
+NODE1ID=$(curl -L https://$IP_ADDR:8000/redfish/v1/Systems/node-1 -k -u "foo:foo" | jq -r '.UUID')
+NODE2ID=$(curl -L https://$IP_ADDR:8000/redfish/v1/Systems/node-2 -k -u "foo:foo" | jq -r '.UUID')
+
+NODE1MAC=$(sudo virsh dumpxml node-1 | grep 'mac address' | grep -ioE "([0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}")
+NODE2MAC=$(sudo virsh dumpxml node-2 | grep 'mac address' | grep -ioE "([0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}")
+```
+- The first 2 commands grab the UUID's from Sushy-Tools, the second two commands grab the mac addresses from virsh.
+- If your VMs aren't defined in root's virsh, you may need to remove "sudo" from the NODE1MAC virsh commands.
+
+
+Create the BMH yamls using the virtual machine information
+### Control plane Node
+```
+cat << EOF > ~/vbmc/node1.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bmc-1-credentials
+  namespace: default
+type: Opaque
+data:
+  username: Zm9vCg==
+  password: Zm9vCg==
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: bmc-1
+  namespace: default
+  labels:
+    cluster-role: control-plane
+spec:
+  online: true
+  bootMACAddress: $NODE1MAC
+  bmc:
+    address: redfish-virtualmedia://$IP_ADDR:8000/redfish/v1/Systems/$NODE1ID
+    disableCertificateVerification: true
+    credentialsName: bmc-1-credentials
+EOF
+```
+### Worker Node
+```
+cat << EOF > ~/vbmc/node2.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bmc-2-credentials
+  namespace: default
+type: Opaque
+data:
+  username: Zm9vCg==
+  password: Zm9vCg==
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: bmc-2
+  namespace: default
+  labels:
+    cluster-role: worker
+spec:
+  online: true
+  bootMACAddress: $NODE2MAC
+  bmc:
+    address: redfish-virtualmedia://$IP_ADDR:8000/redfish/v1/Systems/$NODE2ID
+    disableCertificateVerification: true
+    credentialsName: bmc-2-credentials
+EOF
+```
+
+Copy the baremetal node YAMLs to the metal3-core VM
+```
+scp node*.yaml metal@192.168.125.99:
+```
+
+SSH Into the metal3-core VM
+```
+ssh metal@192.168.125.99
+```
+
+Install Python Dependency
+```
+sudo zypper install -y python310-dbm
+```
+- To use "baremetal" commands, this dependency recently became necessary.
+
+Apply the bare metal node YAML files
+```
+kubectl apply -f node1.yaml
+kubectl apply -f node2.yaml
+```
+- You can monitor the progress of the provisioning using the following commands:
+  - `baremetal node list`
+  - `kubectl get bmh`
+- A `manageable` state is the desired state. It may take a few minutes for provisioning to complete.
```

Comment:
Nit - `baremetal` usage is somewhat confusing as it is showing `manageable` straight away. What I usually do is skip the `python310-dbm` dependency and just `$ watch kubectl get bmh` until their state is `available`.

Reply:
Will change over to kubectl get bmh to prevent seeing managable too early. Also baremetal has a "baremetal show UUID" command that tends to be very helpful for debugging, I'll note that as a troubleshooting step.